frontend/src/hooks/api/getters/useInsights/useInsights.ts

```diff
@@ -27,6 +27,7 @@ export const useInsights = (
     return {
         insights:
             data ||
+            /* @ts-expect-error FIXME: lifecycle trends */
```

Comment:
CC @thomasheartman @sjaanus

Reply:
Aye, aye! 🫡 Not using data from the hook yet, but we'll get there. For now, let's also add the flag name, so that at least we'll catch it by then.
```suggestion
            /* @ts-expect-error FIXME (lifecycleMetrics): lifecycle trends */
```